server/connection_manager.cpp

```diff
@@ -127,25 +133,36 @@ static bool CommandProcessor(int socket, string & s, void * dacs, int ndacs)
 					(*result)[string("when")] = acs[i]->TimeCode();
 					LOG2(_log_, nullptr, VERBOSE);
 				}
-				if (!sr.SerializeToString(&s))
-						throw LOG2(_log_, "t_c could not serialize", FATAL);
+				if (!sr.SerializeToString(&s)) {
+					s = internal_error;
+					LOG2(_log_, fts, FATAL);
+				}
 				LOG2(_log_, nullptr, VERBOSE);
 				SendPB(s, socket);
 			}
 			break;
 
 			case Type::TRACK_COUNT:
 			{
+				// If it should come to pass that there is both a DB error
```

Comment:
Sure, that's fine with me... "Should never happen" :stuck_out_tongue_winking_eye: